<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvToolRigidReg</class>
 <widget class="QWidget" name="vvToolRigidReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Manual</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_28">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Rotation Center(X,Y,Z)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Xval"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Yval"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Zval"/>
           </item>
           <item>
            <widget class="QPushButton" name="resetbutton">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QLabel" name="ytans">
             <property name="text">
              <string>X Trans</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_37"/>
           </item>
           <item>
            <widget class="QSlider" name="xtrans_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="xtrans_sb"/>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_32">
           <item>
            <widget class="QLabel" name="yrot">
             <property name="text">
              <string>Y Rot   </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_40"/>
           </item>
           <item>
            <widget class="QSlider" name="yrot_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="yrot_sb"/>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <item>
            <widget class="QLabel" name="xrot">
             <property name="text">
              <string>X Rot   </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_42"/>
           </item>
           <item>
            <widget class="QSlider" name="xrot_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="xrot_sb"/>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_34">
           <item>
            <widget class="QLabel" name="zrot">
             <property name="text">
              <string>Z Rot   </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_43"/>
           </item>
           <item>
            <widget class="QSlider" name="zrot_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="zrot_sb"/>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="3">
            <widget class="QTextEdit" name="textEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="loadbutton">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="savebutton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QTextEdit" name="textEdit_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Initial Transform</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Current Transform</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_31">
           <item>
            <widget class="QLabel" name="ztrans">
             <property name="text">
              <string>Z Trans</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_39"/>
           </item>
           <item>
            <widget class="QSlider" name="ztrans_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="ztrans_sb"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <widget class="QLabel" name="ytrans">
             <property name="text">
              <string>Y Trans</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_38"/>
           </item>
           <item>
            <widget class="QSlider" name="ytrans_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="ytrans_sb"/>
           </item>
          </layout>
         </item>
         <item row="9" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_49"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Automatic</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>721</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_45"/>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_46"/>
        </item>
        <item row="0" column="3">
         <widget class="QTextEdit" name="tab2textedit"/>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
        <item row="0" column="2">
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="page_3">
           <widget class="QPushButton" name="tab2applybutton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>228</y>
              <width>90</width>
              <height>32</height>
             </rect>
            </property>
            <property name="text">
             <string>Run!</string>
            </property>
           </widget>
           <widget class="QPushButton" name="tab2loadbutton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>183</y>
              <width>91</width>
              <height>32</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_rigid">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>113</y>
              <width>87</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Rigid</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_deformable">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>143</y>
              <width>101</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Deformable</string>
            </property>
           </widget>
           <widget class="QPushButton" name="tab2savebutton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>273</y>
              <width>91</width>
              <height>32</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_transform">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>90</y>
              <width>151</width>
              <height>26</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Identity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Translations</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rigid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Affine</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="translabel">
            <property name="geometry">
             <rect>
              <x>204</x>
              <y>70</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Transform</string>
            </property>
           </widget>
           <widget class="QLabel" name="interpollabel">
            <property name="geometry">
             <rect>
              <x>201</x>
              <y>120</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Interpolator</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_interpolator">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>140</y>
              <width>151</width>
              <height>26</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Nearest Neighborhood</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Linear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BSpline</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BLUT</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="optimlabel">
            <property name="geometry">
             <rect>
              <x>204</x>
              <y>170</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Optimizer</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_optimizer">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>190</y>
              <width>151</width>
              <height>26</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Simplex</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Powell</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FRPR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Regular Step GD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>VersorRigid3D</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Conjugated Gradient</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>L-BFGS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>L-BFGS-B</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="metriclabel">
            <property name="geometry">
             <rect>
              <x>218</x>
              <y>220</y>
              <width>60</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Metric</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_metric">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>240</y>
              <width>151</width>
              <height>26</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Mean Squares</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Normalized CC</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Histogram CC</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gradient Difference</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Viola Wells MI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Histogram MI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mattes MI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Normalized MI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SSD (BLUT FFD)</string>
             </property>
            </item>
           </widget>
           <widget class="QComboBox" name="cb_selectoutput">
            <property name="geometry">
             <rect>
              <x>162</x>
              <y>294</y>
              <width>151</width>
              <height>26</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Transformed Image</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Checker After Reg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Checker Before Reg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Difference After Reg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Difference Before Reg</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="outputlabel">
            <property name="geometry">
             <rect>
              <x>195</x>
              <y>274</y>
              <width>91</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Select Output</string>
            </property>
           </widget>
           <widget class="QLabel" name="presetlabel">
            <property name="geometry">
             <rect>
              <x>208</x>
              <y>20</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Presets</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_presets">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>40</y>
              <width>151</width>
              <height>26</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Mono Modal Registration</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Multi Modal Registration</string>
             </property>
            </item>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../vvIcons.qrc"/>
 </resources>
 <connections/>
</ui>
